native/clij2fft/cppbuild.sh

```diff
@@ -28,7 +28,7 @@ case $PLATFORM in
                -DCMAKE_INSTALL_PREFIX="../../../lib/macosx/" \
                -DCMAKE_CXX_COMPILER="g++" \
                -DCMAKE_CUDA_HOST_COMPILER="g++" \
-		-DCLFFT_LIBRARY_DIR="../../../lib/macosx/" ..
+		-DCLFFT_LIBRARY_DIR="/Users/haase/Downloads/clfft-2.12.2-h83d4a3d_1/lib" ..
         ;;
```

Comment:
Is it possible to make the location generic ?? Instead of '/Users/haase/Downloads/' use a folder that would be on every Mac system?  For example I use 'C:/OpenCL/' on windows, '/opt/OpenCL/' on Linux.

Reply:
Hm. I'm also not a Mac person. The thing is, I apparently downloaded the filder from the internet. Not even sure from where though...

Reply:
I got it from here, likely: https://anaconda.org/conda-forge/clfft/files

Reply:
Hi @haesleinhuepf 

I meant that after downloading the file you could put it in a system directory that would be on every Mac, instead of your user directory.  It would make it easier to build on other computers.